YoCode/PrintToConsole.cs

```diff
@@ -6,90 +6,23 @@ namespace YoCode
 {
     public class PrintToConsole : IPrint
     {
+        string textToPrint;
 
-        public PrintToConsole()
+        public bool AddNewLine(string text)
         {
-            PrintIntroduction();
+            textToPrint += text + Environment.NewLine;
+            return true;
         }
 
-        public void PrintIntroduction()
+        public void PrintMessage()
         {
-            Console.WriteLine(YoCode.messages.Welcome);
+            Console.Write(textToPrint);
+            textToPrint = null;
```

Comment:
perhaps this method should start by checking if textToPrint is null and only ConsoleWrite if it isn't?